landing/introduction.mdx

```diff
@@ -7,52 +7,43 @@ iconType: "light"
-- **[BTC Connect](../api-reference/btc):** Control a Smart Account on an EVM-compatible L2 using your Bitcoin wallet.
+Particle Network current product suite features numerous SDKs made to simplify authentication and onboarding:
 
 <CardGroup cols={2}>
+  <Card title="Particle Auth" icon="shield" href="../api-reference/auth">
+     Onboard users with social authentication using popular platforms like Google, X, and so on.
+   </Card>
  <Card title="Particle Connect" icon="link" href="../api-reference/connect">
- Connect to Web 2.0 accounts and mainstream wallets, enhancing accessibility through a unified interface.
- </Card>
- <Card title="Particle Auth" icon="shield" href="../api-reference/auth">
- Onboard users with one-click authentication using popular platforms like Google, X, and many more.
- </Card>
- <Card title="Embedded Wallet" icon="wallet" href="../api-reference/wallet">
- Integrate a Web3 wallet directly into your dApp, allowing users to manage their assets without leaving your application.
+   Unified modal aggregating both social logins (through Particle Auth) and wallet connection.
  </Card>
  <Card title="Account Abstraction" icon="credit-card" href="../api-reference/aa">
- Modularize your dApp into ERC-4337 AA, leveraging Particle’s Bundler, Paymaster, and compatibility with Smart Account implementations.
+  Full-stack account abstraction SDK driving the implementation of smart accounts, including features like gasless transactions and session keys.
  </Card>
+  <Card title="BTC Connect" icon="btc" href="../api-reference/btc">
+     Connect Bitcoin-native wallets, such as UniSat or Xverse, to applications built on EVM-compatible chains.
+   </Card>
 </CardGroup>
- <Card title="BTC Connect" icon="btc" href="../api-reference/btc">
- Control a Smart Account on an EVM-compatible L2 using your Bitcoin wallet.
- </Card>
 
 ## Quickstart
 
-  <Card title="Integrate social logins" icon="circle-play" href="/resources/quickstart/social-logins">
-    Learn how to integrate social logins in a Next.js project
+  <Card title="Integrate Wallet-as-a-Service" icon="code" href="resources/quickstart/waas/web-quickstart">
+    5-minute walkthrough on utilizing Particle Network's Wallet-as-a-Service to onboard users through social logins.
   </Card>
 
 
 ## Connect with Particle Network
 
-Stay up to date with Particle Network and reach out to us at any time for questions, issues, and feedback.
+Stay up to date with Particle Network. Reach out to us anytime with questions, issues, or feedback.
 
 <CardGroup cols={3}>
-  <Card title="Particle Network Twitter" icon="twitter" href="https://x.com/ParticleNtwrk">
-    Follow us on Twitter to stay updated with the latest news and announcements.
+  <Card title="@ParticleNtwrk" icon="x-twitter" href="https://x.com/ParticleNtwrk">
```

Comment:
Not too sure about these; saying "Main X" "Developer X" or "Main Twitter" "Developer Twitter" felt off. Perhaps there's a better way to do this